core/drivers/pl022_spi.c

```diff
@@ -123,8 +123,10 @@
 #define SSPMIS_RTMIS	SHIFT_U32(1, 1)
 #define SSPMIS_RORMIS	SHIFT_U32(1, 0)
 
-#define SSPICR_RTIC	SHIFT_U32(1, 1)
-#define SSPICR_RORIC	SHIFT_U32(1, 0)
+#define SSPICR_RTIC		SHIFT_U32(1, 1)
+#define SSPICR_RTIC_CLEAR	SHIFT_U32(1, 1)
+#define SSPICR_RORIC		SHIFT_U32(1, 0)
+#define SSPICR_RORIC_CLEAR	SHIFT_U32(1, 0)
```

Comment:
minor: seems useless duplications of `SSPICR_RTIC` and `SSPICR_RORIC`, these are already 'clear signal' bit fields.